test/e2e/helpers.js

```diff
@@ -745,7 +745,7 @@ const PRIVATE_KEY =
   '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC';
 
 const PRIVATE_KEY_TWO =
-  '0xa444f52ea41e3a39586d7069cb8e8233e9f6b9dea9cbb700cce69ae860661cc8';
+  '0xf444f52ea41e3a39586d7069cb8e8233e9f6b9dea9cbb700cce69ae860661cc8';
```

Comment:
Corrected the private key two to ensure that we create the public address specified as ACCOUNT_2, rather than using public address 0x66ba6a97bf7e686c171cae1b754bca1521888dc1 (which is what the `a` gave us.